cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -306,6 +333,67 @@ message AvailablePackageDetail {
   google.protobuf.Any custom_detail = 13;
 }
 
+// InstalledPackageSummary
+//
+// An InstalledPackageSummary provides a summary of an installed package
+// useful when aggregating many installed packages.
+message InstalledPackageSummary {
+  // TODO: add example for API docs
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
+
+  // InstalledPackageReference
+  //
+  // A reference uniquely identifying the package.
+  InstalledPackageReference installed_package_ref = 1;
+
+  // Name
+  //
+  // A name given to the installation of the package (eg. "my-postgresql-for-testing").
+  string name = 2;
+
+  // PkgVersionReference
```

Comment:
PkgVersionReference => VersionReference?

Reply:
This is the public name of the field on the go struct, not the type. It's confusing, but since the name in the protobuf below is `pkg_version_reference`, the corresponding Go struct is generated with go-naming conventions as `PkgVersionReference`. You can see that in the generated changes to cmd/kubeapps-apis/gen/core/packages/v1alpha1/packages.pb.go above.

Aside: So far, we have been documenting each field using the name of the field in the go struct, but as you'd expect, for the generated JS/Swagger, it's generated with JS naming conventions as `pkgVersionReference`, which you can see in the swagger changes above. The title for the swagger documentation is taken from this comment, so appears as `PkgVersionReference`. Perhaps we want to switch all our titles here to use the JS names for the swagger docs. Anyway, not something for here, imo.